packages/rancher-monitoring/rancher-monitoring.patch

```diff
@@ -2026,11 +2026,12 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/val
  
  ## Provide a k8s version to auto dashboard import script example: kubeTargetVersionOverride: 1.16.6
  ##
-@@ -76,8 +334,18 @@
+@@ -76,8 +334,19 @@
  
  ##
  global:
-+  systemDefaultRegistry: ""
++  cattle:
```

Comment:
I think you are missing `global.cattle.clusterId` and `global.cattle.clusterName` (In all other values.yaml as well)

Reply:
we only need to accept those for charts that utilize these values right? because backup-restore-operator chart doesn't have them either

Reply:
I agree with Rajashree; I think these fields could be added in within a future PR, but since the Charts don't support any changes based on these fields at the moment I think it should be safe to omit it.

Reply:
@aiyengar2 @mrajashree I was going based on a comment in 2dotx-dev, but I wasn't part of the design meeting where this was discussed so if that's the case then LGTM.